ui/client/components/graph/NodeDetailsContent.js

```diff
@@ -173,15 +179,15 @@ export class NodeDetailsContent extends React.Component {
       case 'SubprocessOutputDefinition':
         return (
           <div className="node-table-body">
-            {this.createField("input", "Id", "id", true)}
-            {this.createField("input", "Output name", "outputName")}
+            {this.createField("input", "Id", "id", true, true)}
```

Comment:
Shouldn't this be more general solution? We could pass list of validators, in most cases predefined 1-element list with "notEmptyValidator", later we can put there a validator extracting value from processValidation response (#233). I think there should be single way od putting error information below fields. Later we have to consider if several messages can be presented below a field (I think it is a case, processValidation allows it). And how to distinguish between validations that prevent saving the modal from ones that "warn" only.

Reply:
Done